spec/lib/wizard/wizard_builder_spec.rb

```diff
@@ -122,88 +114,4 @@ class GlobalPathInstance
     end
   end
 
-  context 'icons step' do
-    let(:icons_step) { wizard.steps.find { |s| s.id == 'icons' } }
-
-    it 'should set the right default value for the fields' do
-      upload = Fabricate(:upload)
-      upload2 = Fabricate(:upload)
-
-      SiteSetting.favicon = upload
-      SiteSetting.large_icon = upload2
-
-      fields = icons_step.fields
-      favicon_field = fields.first
-      large_icon_field = fields.last
-
-      expect(favicon_field.id).to eq('favicon')
-      expect(favicon_field.value).to eq(GlobalPathInstance.full_cdn_url(upload.url))
-      expect(large_icon_field.id).to eq('large_icon')
-      expect(large_icon_field.value).to eq(GlobalPathInstance.full_cdn_url(upload2.url))
-    end
-  end
-
-  context 'introduction step' do
-    let(:wizard) { Wizard::Builder.new(moderator).build }
-    let(:introduction_step) { wizard.steps.find { |s| s.id == 'introduction' } }
-
-    context 'step has not been completed' do
-      it 'enables the step' do
-        expect(introduction_step.disabled).to be_nil
-      end
-    end
-
-    context 'step has been completed' do
-      before do
-        wizard = Wizard::Builder.new(moderator).build
-        introduction_step = wizard.steps.find { |s| s.id == 'introduction' }
-
-        # manually sets the step as completed
-        logger = StaffActionLogger.new(moderator)
-        logger.log_wizard_step(introduction_step)
-      end
-
-      it 'disables step if no welcome topic' do
-        expect(introduction_step.disabled).to eq(true)
-      end
-
-      it 'enables step if welcome topic is present' do
-        topic = Fabricate(:topic, title: 'Welcome to Discourse')
-        welcome_post = Fabricate(:post, topic: topic, raw: "this will be the welcome topic post\n\ncool!")
-
-        expect(introduction_step.disabled).to be_nil
-      end
-    end
-  end
-
-  context 'privacy step' do
-    let(:privacy_step) { wizard.steps.find { |s| s.id == 'privacy' } }
-
-    it 'should set the right default value for the fields' do
```

Comment:
I think it'd be useful to keep this test and update it so it asserts that the new fields (e.g. `invite_only`, `must_approve_users` etc.) have the right default values (unless this is already tested elsewhere).

Reply:
Added back and updated the test.